steps/swiftlint/0.4.1/step.yml

```diff
@@ -58,7 +58,7 @@ inputs:
     - junit
     - emoji
   reporter: xcode
-- opts:
+- opts: "no"
```

Comment:
you "fixed" this at the wrong place ;)

Reply:
Dooh... Think I'm going to bed soon... I'm an old man... 👴

Reply:
Don't worry, happens with the bests too ;)